### PR TITLE
Some ui fix and hide

### PR DESCRIPTION
**Pull request by FanchTheSystem**
https://gitlab.com/ifb-elixirfr/ansible-roles/ansible-my/issues/40
https://gitlab.com/ifb-elixirfr/ansible-roles/ansible-my/issues/3

### CODE REVIEW

**Comment by osallou** [comment]
Don't understand the unsubscribe, if user is not active he is not subscribed anymore

**Comment by FanchTheSystem** [comment]
> Don't understand the unsubscribe, if user is not active he is not subscribed anymore

It is to disable unsubscribe button for active user (as at ifb we don't want to allow people to unsubscribe from the cluster mailing list)

**Comment by osallou** [comment]
config.newsletters etc... are hardcoded value for now, not taken from config file

File: manager2/src/app/auth/register/register.component.html [hunk 1]
```diff
@@ -18,19 +18,19 @@
...
                 <div class="col-md-4">
-                    <label for="email">User id</label>
+                    <label for="email">Username</label>
                     <input autocomplete="username" [ngModelOptions]="{standalone: true}" [(ngModel)]="userid" type="text" id="userid" class="form-control" required maxlength="12"/>
 
                 </div>
             </div>
             <div class="form-group">
-                <label for="email">Professional email</label>
+                <label for="email">Professional academic email</label>
```
Comment (osallou) [inline manager2/src/app/auth/register/register.component.html]: why academic ? We may have non academic

Comment (FanchTheSystem) [reply manager2/src/app/auth/register/register.component.html]: you're right 74b9261e5120bfcd165953309a9d4ce535ef70ca

**Comment by FanchTheSystem** [comment]
> config.newsletters etc... are hardcoded value for now, not taken from config file

sorry I don't understand, what I need to do (or not) ? ;)

**Pull request merged**